topics/topic07/index.html

```diff
@@ -0,0 +1,68 @@
+<html>
+
+<head>
+    <script src="https://unpkg.com/fabric@5.0.0/dist/fabric.min.js"></script>
+    <script src="../../../utils/geoutil.js"></script>
+</head>
+
+<body>
+    <h1>Isotomic Conjugate Points</h1>
+    <div>
+        <span>Resources:</span>
+        <ul>
+            <li>
+                <h3 color="black"><a href=https://www.math.uci.edu/~gdp/projects/topics/topic07.pdf >Isogonal Conjugate and Isotomic Conjugate Points</a></h3>            </li>
+            <li>
+            
+                <div class="app">
+                    <svg id="tri-app_isotomic" width="500" height="500" style="border: 2px solid black">
+            
+            
+                        <polygon id="tri_isotomic"  stroke="black" fill="white"/>
+                                    
+                            <line id="lineA_isotomic"  stroke="purple" />
```

Comment:
nit: try use the key combination in VSCode: option + shift + f; this reformat the file which improves readability and makes the code format consistent